watertap/property_models/anaerobic_digestion/adm1_properties_vapor.py

```diff
@@ -310,13 +309,27 @@ def p_sat_rule(b, j):
                 )
 
         self._p_sat = pyo.Constraint(
-            self.params.solute_set, rule=p_sat_rule, doc="P for not solutes"
+            self.params.solute_set,
+            rule=p_sat_rule,
+            doc="Saturation pressure for components",
         )
 
-        def p_w_sat_rule(b):
-            return self.p_w_sat == 0.0557 * 101325 * pyo.units.Pa
+        def p_w_sat_rule(b, t):
+            return (
+                self.p_w_sat
+                == 0.0313
+                * pyo.exp(
+                    5290
```

Comment:
Do we want to define 0.0313 and 5290 as parameters since it's not immediately clear what these numbers represent/where they come from (or maybe it's obvious and I'm just unaware)?